src/test/java/com/bobocode/hoverla/bring/BringApplicationTest.java

```diff
@@ -0,0 +1,71 @@
+package com.bobocode.hoverla.bring;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import com.bobocode.hoverla.bring.context.ApplicationContext;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class BringApplicationTest {
+    private final String PACKAGE = "com.bobocode.hoverla.bring";
+
+    @DisplayName("Throws IllegalArgumentException: no packages were provided")
+    @Test
+    void throwsExceptionPackageToScanIsEmpty() {
+        String exceptionMessage = "Argument [packagesToScan] must contain at least one element";
+        var builder =  BringApplication.getContextBuilder();
+        assertThatThrownBy(builder::build)
```

Comment:
I guess builder is what you want to test separately, avoiding several actions and checks in one test case

Reply:
@DanielChanger divided into two separate classes